src/discussions/posts/post/Post.jsx

```diff
@@ -81,31 +106,44 @@ function PostHeader({
           </span>
         </div>
       </div>
-      <div className="d-flex mr-3">
-        <Icon src={QuestionAnswer} />
-        <span style={{ minWidth: '2rem' }}>
-          {post.commentCount}
-        </span>
-      </div>
+      {preview
+        ? (
+          <div className="d-flex mr-3">
+            <Icon src={QuestionAnswer} />
+            <span style={{ minWidth: '2rem' }}>
+              {post.commentCount}
+            </span>
+          </div>
+        )
+        : <ActionsDropdown commentOrPost={post} onCommentAction={handleActions} />}
     </div>
   );
 }
 
 PostHeader.propTypes = {
   intl: intlShape.isRequired,
   post: postShape.isRequired,
+  preview: PropTypes.bool.isRequired,
 };
 
 function Post({
   post,
+  preview,
   intl,
 }) {
   const dispatch = useDispatch();
 
   return (
     <div className="d-flex flex-column p-2.5 w-100">
-      <PostHeader post={post} intl={intl} />
-      <div className="d-flex mt-2 mb-0 p-0" dangerouslySetInnerHTML={{ __html: post.renderedBody }} />
+      <PostHeader post={post} intl={intl} preview={preview} />
+      <div
+        className="d-block mt-2 mb-0 p-0 overflow-hidden text-break"
+        dangerouslySetInnerHTML={{ __html: post.renderedBody }}
+        style={{
```

Comment:
Good opportunity to use `classnames` here. I think the lib is already plugged in.

Reply:
It isn't currently in use by this repo, but will add it.

Reply:
More context: 
MFEs just depend on classnames directly. Because we distribute Paragon as a source, it generally won't increase bundle size assuming the versions don't differ wildly. The MFE and paragon will likely use the same node_modules/classnames code when the MFE is bundled.